### PR TITLE
More Hyperscale Colours!

### PR DESCRIPTION
**Pull request by Wirpy**
## About The Pull Request

I felt like colours I'd like to use for armour and clothing customization were missing and the ones we have currently were not the hues / tones and what not that I were looking for and furthermore I have seen people voice some similiar concerns so I've made some more hyperscale colours. 

1a7f332742e528e9f50e00e301ab6525

List left from right;
Nightshade (Black default armor)
Space cadet (Blue)
Dark blue (Blue with purple hue/tones+darker)
Syndie red (Red)
Syndie Black (Default armor / blackish grey)
Elite orange (Orange visor)
Syndie green (Green visor)

c4183f6e0c2c7465313e82bcc11bc101

Approved by Kuro, very nice!

## Why It's Good For The Game

Increases the capabiltiies for players to customize their armour and clothing with new exciting colours ergo more customization good!!

## Changelog
:cl: wirpy
add: Bunch of new hyperscale colours
add: Nightshade (Black in default armor)
add: Space cadet (Blue)
add: Dark blue (BLue)
Add: Syndie Red (Red)
Add: Syndie Black (Black in default armor)
Add: Elite orange (Orange visor)
Add: Syndie green (Green visor)
/:cl:

### CODE REVIEW

File: code/__DEFINES/modular_armor.dm [hunk 1]
```diff
@@ -31,6 +31,8 @@
 #define ARMOR_PALETTE_NIGHT_SLATE "#1F2228#252C3D#3A3C4F#4E546B#7D8A91"
 #define ARMOR_PALETTE_FALL "#2F200F#4D3927#7A5A3E#A87C55#BF9E81"
 #define ARMOR_PALETTE_STEEL "#121921#2b3541#444a4f#5f6260#8a8e8b"
+#define ARMOR_PALETTE_Syndie_Black "#181719#232226#3e3d3f#545350#63615e"
```
Comment (XSlayer300) [inline code/__DEFINES/modular_armor.dm:34]:
```suggestion
#define ARMOR_PALETTE_SYNDIE_BLACK "#181719#232226#3e3d3f#545350#63615e"
```
stop

File: code/__DEFINES/modular_armor.dm [hunk 2]
```diff
@@ -144,6 +151,8 @@
 			"Night Slate" = ARMOR_PALETTE_NIGHT_SLATE,\
 			"Fall" = ARMOR_PALETTE_FALL,\
 			"Steel" = ARMOR_PALETTE_STEEL,\
+			"Nightshade" = ARMOR_PALETTE_NIGHTSHADE,\
+			"Syndie Black" = ARMOR_PALETTE_Syndie_Black,\
```
Comment (XSlayer300) [inline code/__DEFINES/modular_armor.dm:155]:
```suggestion
			"Syndie Black" = ARMOR_PALETTE_SYNDIE_BLACK,\
```

**Comment by Wirpy** [comment]
@XSlayer300 fixed, prob had a brainfart or something while doing that one lol

**Comment by Kuro020** [comment]
Lazy ass code maints

**Pull request merged**